xskillscore/core/probabilistic.py

```diff
@@ -475,6 +478,64 @@ def threshold_brier_score(
         return res.mean(dim, keep_attrs=keep_attrs)
 
 
+def _check_bin_dim(ds):
+    """Assert that bin_dim is in ds. Try to guess and rename edges dimension."""
+    bin_dim = "category_edge"
+    if bin_dim not in ds.dims:
+        raise ValueError(f"require {bin_dim} dimension, found {ds.dims}")
+    return ds
+
+
+def _check_data_within_edges(forecasts, forecasts_edges):
```

Comment:
Delete as we now have internally -inf as lower edge and +inf as upper edge we don’t need to calc.